.github/ISSUE_TEMPLATE/release.md

```diff
@@ -9,23 +9,12 @@ assignees: ''
 
 ## 1. Preparation on Thursday
 
-- [ ] Merge pull requests in the [Ready To Go] column
-- [ ] Include translations
-  <details><summary>Command line instructions:</summary>
-    <pre>
-    <code>
-    git checkout master
-    git pull upstream master
-    tx pull --force
-    git commit -a -m "Update all locales with the latest Transifex translations"
-    git push upstream master
-    </code>
-    </pre>
-  </details>
-- [ ] Create a tag:
-    - `script/tag_release` will auto increment patch version, otherwise
+- [x] Merge pull requests in the [Ready To Go] column
+- [x] Include translations: `script/release/udpate_locales`
+- [x] Create a tag:
+    - `script/release/tag` will auto increment patch version, otherwise
     - `git push upstream HEAD:refs/tags/vX.Y.Z`
-- [ ] [Draft new release]. Look at previous [releases] for inspiration.
+- [x] [Draft new release]. Look at previous [releases] for inspiration.
```

Comment:
I wouldn't pre-tick any of these items either. I would combine all items covered by the script into one task though. So then it's only one tick.

There are two flows now. You can either create the issue through Github or run the script first and then follow the link to create the issue. The latter has the gotcha that you need to remember to merge Ready To Go PRs first.

The other question is: what do we do with minor and major releases? You often only know that ones you draft the release notes which happens after the tagging. Do we then remove the tag again and create a new one?

Reply:
> The latter has the gotcha that you need to remember to merge Ready To Go PRs first.

I think we could add a reminder at the start of the the script for that. 
But..

> The other question is: what do we do with minor and major releases? You often only know that ones you draft the release notes which happens after the tagging. Do we then remove the tag again and create a new one?

That's a good question. Perhaps I'm automating too much. I think for now, I'll remove the shortcut link for creating the issue, until we sort that out.

Reply:
I have to admit, I'm not always thinking about what version level it should be. Maybe we could introduce more explicit rules. Eg if a PR with migration deleting data is tagged as `data destructive`, we could systematically say that means it's a minor increment. 🤔 

But I think we don't want to spend time thinking more about it..

Reply:
>  I'm not always thinking about what version level it should be.

Me neither. I looked at past releases before but forgot the result again. So this time I documented it:
https://community.openfoodnetwork.org/t/release-numbering/105/6